presale/views/presale_order_views.xml

```diff
@@ -0,0 +1,78 @@
+<odoo>
+    <record id="presale_order_view_tree" model="ir.ui.view">
+        <field name="name">presale.order.tree</field>
+        <field name="model">presale.order</field>
+        <field name="arch" type="xml">
+            <tree string="Presales Orders">
+                <field name="name"/>
+                <field name="customer_id"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="presale_order_type_view_form" model="ir.ui.view">
+        <field name="name">presale.order.form</field>
+        <field name="model">presale.order</field>
+        <field name="arch" type="xml">
+            <form string="Order">
+                <header>
+                    <button
+                        name="action_validate"
+                        string="Validate"
+                        type="object"
+                        icon="fa-check"
+                        attrs="{'invisible': [('state', '=', 'confirmed')]}"
+                    />
+                    <button
+                        type="action"
+                        name="%(presale.sale_order_action)d"
+                        class="oe_stat_button"
+                        icon="fa-dollar"
+                        attrs="{'invisible': [('state', '=', 'draft')]}"
+                    >Sales Orders</button>
+                    <field name="state" widget="statusbar" statusbar_visible="draft, confirmed">Sale
+                        Order</field>
+                </header>
+                <sheet>
+                    <h1>
+                        <field name="name"/>
+                    </h1>
+                    <group>
+                        <group>
+                            <field name="customer_id"/>
+                        </group>
+                        <notebook>
+                            <page string="Order lines">
+                                <field name="order_line_ids"/>
+                            </page>
+                        </notebook>
+                    </group>
+                </sheet>
+
+                <div class="oe_chatter">
+                    <field name="message_follower_ids" groups="base.group_user"/>
+                    <field name="activity_ids"/>
+                    <field name="message_ids"/>
+                </div>
+            </form>
+        </field>
+    </record>
+
+    <record id="presale_order_type_search" model="ir.ui.view">
+        <field name="name">presale.order.search</field>
+        <field name="model">presale.order</field>
+        <field name="arch" type="xml">
+            <search string="Presales Order">
+                <field name="name" string="Order name"/>
+            </search>
+        </field>
+    </record>
+
+    <record id="presale_order_action" model="ir.actions.act_window">
+        <field name="name">Presales Orders</field>
+        <field name="res_model">presale.order</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+</odoo>
```

Comment:
it's better to move the action code to a seperate file under `data/` since they doesn't represent views (you have 2 other actions in other files)